discord_slash/utils/manage_components.py

```diff
@@ -211,6 +212,7 @@ def create_select(
     :param placeholder: Custom placeholder text if nothing is selected
     :param min_values: The minimum number of items that **must** be chosen
     :param max_values: The maximum number of items that **can** be chosen
+    :param disabled: Disable this component, default False
```

Comment:
`Disables this component. Defaults to ``False``. `

Reply:
Could you make this a suggestion so I can just hit accept

Reply:
yep done